### PR TITLE
added dummydata recipe

### PR DESCRIPTION
**Pull request by bulli92**

### CODE REVIEW

**Comment by bulli92** [comment]
mmh, any idea what could cause the lint problem? Can not identify any issue with meta.yml

**Comment by bulli92** [comment]
Don't understand why this does not install on travis. The log looks o.k., only the import seems not to work. I am puzzled here, as it works easily on my machines and the package is not very complicated at all.

Can somebody help?

File: recipes/dummydata/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  skip: True  # [win]
+  skip: True  # [py>27]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - numpy x.x
+    - netcdf4
+    - setuptools
```
Comment (ericdill) [inline recipes/dummydata/meta.yaml]: only setuptools and python are build requirements

File: recipes/dummydata/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  run:
+    - python
+    - numpy x.x
```
Comment (ericdill) [inline recipes/dummydata/meta.yaml]: drop the `x.x` after numpy. You only need to include `x.x` after numpy in build and run requirements if you're linking against the numpy C API, which this package is not.

File: recipes/dummydata/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    - netcdf4
```
Comment (ericdill) [inline recipes/dummydata/meta.yaml]: Add python-dateutil as a run dependency

File: recipes/dummydata/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
+{% set name = "dummydata" %}
+{% set version = "0.1.2" %}
```
Comment (ericdill) [inline recipes/dummydata/meta.yaml]: probably want to bump this to 0.1.3 now

File: recipes/dummydata/meta.yaml (hunk 3 above)
Comment (bulli92) [inline recipes/dummydata/meta.yaml]: done

File: recipes/dummydata/meta.yaml (hunk 2 above)
Comment (bulli92) [inline recipes/dummydata/meta.yaml]: done

File: recipes/dummydata/meta.yaml (hunk 1 above)
Comment (bulli92) [inline recipes/dummydata/meta.yaml]: done

File: recipes/dummydata/meta.yaml (hunk 4 above)
Comment (bulli92) [inline recipes/dummydata/meta.yaml]: did set to 0.1.4

**Comment by bulli92** [comment]
o.k., seems to work now!
Thanks for the comments.

**Pull request merged**